Modules/_testcapimodule.c

```diff
@@ -144,6 +144,67 @@ test_sizeof_c_types(PyObject *self, PyObject *Py_UNUSED(ignored))
 #endif
 }
 
+static PyObject*
+test_gc_control(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    int orig_enabled = PyGC_IsEnabled();
+    const char* msg = "ok";
+    int old_state;
+
+    old_state = PyGC_Enable();
+    msg = "Enable(1)";
```

Comment:
Bugs are unlikely. IMO using assert() is good enough for _testcapi tests. You can leave the code as it is ;-)

See for example test_refcount_funcs() which uses assert().

Reply:
I was thinking that assertions can be compiled out, which is decidedly not intended here.

Reply:
In general yes, but _testcapimodule.c starts with:

```
/* Always enable assertions */
#undef NDEBUG
```

Reply:
Good to know! I'd keep the test as it is though. It doesn't really hurt, I think.